### ISSUE TITLE
Prohibitive license conuterproductive to educational materials

### ISSUE THREAD

**Issue by e12e** [firasd/react-server-tutorial#1]
Hi,

thank you so much for the fine tutorial and article.

Unfortunately I'm a more than a little concerned with the license text: "The examples provided by Facebook are for non-commercial testing and evaluation purposes only. Facebook reserves all rights not expressly granted.". (a8c9f89c45def9d54b8a8df5abdf6d846eeba9a0)

This means that using or substantially reproducing your examples and code in production will leave users in breach of copyright! Personally I strongly believe that the best license for tutorial example code is a dual CC0/public domain (where applicable) and a liberal license like BSD/MIT/APL, possibly LGPL (but for example code, I think "no strings attached" is more constructive).

Is there any chance the license could be changed? Then I wouldn't have to be afraid to recommend the article to people that work on production systems...

[edit: I just realized the license is from the upstream Facebook repository, I'll create an issue there as well. ed2: https://github.com/reactjs/react-tutorial/issues/129]

**Comment by firasd** [comment]
Yeah the license is a bit awkward. I didn’t worry about forking it since the example code is so minimal, but let’s see if upstream has any response…
